.evergreen/.evg.yml

```diff
@@ -533,15 +533,19 @@ buildvariants:
 #  display_name: "${version} ${topology} ${auth-and-ssl} ${ruby} ${os}"
 #  tasks:
 #     - name: "test"
-- matrix_name: "rhel70-all-mongodbs-all-rubies-all-topologies-all-auth-and-ssl"
+-
+  #matrix_name: "rhel70-all-mongodbs-all-rubies-all-topologies-all-auth-and-ssl"
+  matrix_name: "all"
   matrix_spec: { auth-and-ssl: "*", ruby: "*", mongodb-version: "*", topology: "*" }
   display_name: "${mongodb-version} ${topology} ${auth-and-ssl} ${ruby}"
   run_on:
     - rhel70-small
   tasks:
      - name: "test"
 
-- matrix_name: "rhel70-latest-all-rubies-all-topologies-all-auth-all-ssl-compression"
+-
+  #matrix_name: "rhel70-latest-all-rubies-all-topologies-all-auth-all-ssl-compression"
+  matrix_name: "3.6-compression"
   matrix_spec: { auth-and-ssl: "*", ruby: "*", mongodb-version: "3.6", topology: "*", compressor: 'zlib' }
```

Comment:
Compression should be tested against 3.6+. I think you can specify an array here of 3.6, 4.0, and "latest".